overlord/hookstate/ctlcmd/restart.go

```diff
@@ -33,8 +33,7 @@ The restart command restarts the given services of the snap. If executed from th
 )
 
 func init() {
-	// FIXME: uncomment once the feature is fixed to work on install/refresh
-	// addCommand("restart", shortRestartHelp, longRestartHelp, func() command { return &restartCommand{} })
+	addCommand("restart", shortRestartHelp, longRestartHelp, func() command { return &restartCommand{} })
```

Comment:
Just a comment, I find it curious we disabled this at the client, not at the daemon.

Reply:
No, this *is* the daemon side. snapctl is the only place where the entire commands, including the help are generated on the daemon side and the client sends a very simple json with argv, literally. The client is implemented in cmd/snapctl.